lib/about_yml/schema.json

```diff
@@ -195,6 +228,21 @@
                     "description": "Anchor text for the link"
                 }
             }
+        },
+        "contact": {
```

Comment:
Note to folks: This is a format that's already been rolling out, so this is a change to catch up with current practice.